src/reducers/locationManagerReducers.js

```diff
@@ -0,0 +1,21 @@
+import { CHANGE_LOCATION, CHANGE_SEARCH_STRING } from '../actions/locationsManagerActions';
+
+const initialState = {
+  currentLocation: 'Gomel',
```

Comment:
Try to remove this hardcode

Reply:
Done. I have changed this initial state to empty object, because this state will got from local storage or from API if local storage item will be empty.